public/app/pages/configuration.js

```diff
@@ -189,6 +194,9 @@ Vue.component('Configuration', {
 			this.time = '08 =00';
```

Comment:
Can you fix this to 08:00 please? It's not releated to your work, but when you're on this already :)